src/style.css

```diff
@@ -1 +1,114 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+:root {
+  --main-color: #3d4e61;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+  color: var(--main-color);
+  font-weight: 300;
+}
+
+.search {
+  display: flex;
+  justify-content: center;
+  margin: 20px auto;
+}
```

Comment:
Be consistent with your margins. According to the checklist, you should add only top or bottom margins, but not both to avoid potential margin collapse issues. In your case, you are adding a margin to all sides of the `.search` class.